src/lightning_app/utilities/app_helpers.py

```diff
@@ -419,7 +419,7 @@ def _set_level(self):
 
 def _state_dict(flow: "LightningFlow"):
     state = {}
-    flows = [flow] + list(flow.flows.values())
+    flows = [flow] + flow.get_all_children()
```

Comment:
Note for another PR. Let's name `get_all_children` to `flows`.

Reply:
+1 

I'm going to revert this, it's not needed here.